src/java/org/apache/cassandra/utils/FBUtilities.java

```diff
@@ -458,15 +463,15 @@ public static String getReleaseVersionString()
     {
         Properties props = getVersionProperties();
         if (props == null)
-            return System.getProperty("cassandra.releaseVersion", UNKNOWN_RELEASE_VERSION);
+            return RELEASE_VERSION.getString();
```

Comment:
I would not put "Unknown" as default in CassandraRelevantProperties but I would do `RELEASE_VERSION.getString(UNKNOWN_RELEASE_VERSION)`. As it is done now, we have "Unknown" at two places - in `CassandraRelevantProperties` as default and the second time as a value in `UNKNOWN_RELEASE_VERSION` constant. Having it once is enough.